source/configure/calls-troubleshooting.md

```diff
@@ -0,0 +1,385 @@
+   ```
+
+   On a client machine:
+   
+   ```bash
+   nc -v YOUR_RTCD_SERVER 8443
+   ```
+
+### Network Packet Analysis
+
+To capture and analyze network traffic:
+
+1. **Capture UDP traffic on the RTCD server**:
+
+   ```bash
+   sudo tcpdump -n 'udp port 8443' -i any
+   ```
+
+2. **Capture TCP API traffic**:
+
+   ```bash
+   sudo tcpdump -n 'tcp port 8045' -i any
+   ```
+
+3. **Analyze traffic patterns**:
+   
+   - Verify packets are flowing both ways
+   - Look for ICMP errors that might indicate firewall issues
+   - Check for patterns of packet loss
+
+4. **Use Wireshark for deeper analysis**:
+   
+   For more detailed packet inspection, capture traffic with tcpdump and analyze with Wireshark:
+   
+   ```bash
+   sudo tcpdump -n -w calls_traffic.pcap 'port 8443'
+   ```
+      
+   Then analyze the `calls_traffic.pcap` file with Wireshark.
+
+### Firewall Configuration Checks
+
+1. **Check iptables rules** (Linux):
+
+   ```bash
+   sudo iptables -L -n
+   ```
+      
+   Ensure there are no rules blocking UDP port 8443 or TCP ports 8045/8443.
+
+2. **Check cloud provider security groups**:
+   
+   Verify that security groups or network ACLs allow:
+   - Inbound UDP on port 8443 from client networks
+   - Inbound TCP on port 8045 from Mattermost server networks
+   - Inbound TCP on port 8443 (if TCP fallback is enabled)
+
+3. **Check intermediate firewalls**:
+   
+   - Corporate firewalls might block UDP traffic
+   - Some networks might require TURN servers for traversal
+
+## Log Analysis
+
+### RTCD Logs
+
+The RTCD service logs important events and errors. Set the log level to "debug" for troubleshooting:
+
+1. **In the configuration file**:
+
+   ```toml
+   [logger]
+   enable_file = true
+   file_level = "DEBUG"
+   ```
+
+   Restart the RTCD service after making these changes
+
+2. **Common log patterns to look for**:
+
+   - **Connection errors**: Look for "failed to connect" or "connection error" messages
+   - **ICE negotiation failures**: Look for "ICE failed" or "ICE timeout" messages
+   - **API authentication issues**: Look for "unauthorized" or "invalid API key" messages
+
+### Mattermost Logs
+
+Check the Mattermost server logs for Calls plugin related issues:
+
+1. **Enable debug logging** in System Console > Environment > Logging > File Log Level
+
+2. **Filter for Calls-related logs**:
+
+   ```bash
+   grep -i "calls" /path/to/mattermost.log
+   ```
+      
+3. **Look for common patterns**:
+   
+   - Connection errors to RTCD
+   - Plugin initialization issues
+   - WebSocket connection problems
+
+### Browser Console Logs
+
+Instruct users to check their browser console logs:
+
+1. **In Chrome/Edge**:
+   - Press F12 to open Developer Tools
+   - Go to the Console tab
+   - Look for errors related to WebRTC, Calls, or media permissions
+
+2. **Specific patterns to look for**:
+   
+   - "getUserMedia" errors (microphone permission issues)
+   - "ICE connection" failures
+   - WebSocket connection errors
```

Comment:
I'm not sure how actionable these would be. Overall, it feels more like internal-facing documentation (for support) rather than something meant for customers.